test/facility-loan-transaction/get-loan-transaction-by-bundle-identifier.api-test.ts

```diff
@@ -0,0 +1,252 @@
+import { PROPERTIES } from '@ukef/constants';
+import { DateStringTransformations } from '@ukef/modules/date/date-string.transformations';
+import { GetFacilityLoanTransactionResponseItem } from '@ukef/modules/facility-loan-transaction/dto/get-loan-transaction-response.dto';
+import { withAcbsAuthenticationApiTests } from '@ukef-test/common-tests/acbs-authentication-api-tests';
+import { IncorrectAuthArg, withClientAuthenticationTests } from '@ukef-test/common-tests/client-authentication-api-tests';
+import { Api } from '@ukef-test/support/api';
+import { ENVIRONMENT_VARIABLES, TIME_EXCEEDING_ACBS_TIMEOUT } from '@ukef-test/support/environment-variables';
+import { GetFacilityLoanTransactionGenerator } from '@ukef-test/support/generator/get-facility-loan-transaction-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import nock from 'nock';
+
+describe('GET /facilities/{facilityIdentifier}/loan-transactions/{bundleIdentifier}', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const dateStringTransformations = new DateStringTransformations();
+  const portfolioIdentifier = PROPERTIES.GLOBAL.portfolioIdentifier;
```

Comment:
We can destructure this

Reply:
Changed to `const { portfolioIdentifier } = PROPERTIES.GLOBAL;`